### ISSUE TITLE
Issue with mod/map list in launcher for in game lobby

### ISSUE THREAD

**Issue by BRNKoINSANITY** [FAForever/client#1067]
I recently purged my maps and mods folders, as there were duplicate and triplicate listings of basically everything in them, and that purge showed the bug that caused the duplicates.

If you download a mod/map then double click on the FAF launcher, you can select that mod/map from the menu, and there is only one.

If you then download a second mod/map and open phantom, there will be a duplicate of the first mod and a single listing of the new mod.

Opening a third game in a new featured mod makes another new copy of each one.

Any further mod/map downloads only appear in the bottom duplicate of the mods list. Each list is complete 1-Z, then starts anew 1-Z.

TL:DR launching from featured mods duplicates your map and mod lists, resulting in literally hundreds of confusing entries in your lists.

**Comment by FlyingThunder** [comment]
is this resolved in #1065 @Wesmania  ?

**Comment by Exotic-Retard** [comment]
should be, i reported the same issue on brnks behalf earlier, i think

**Comment by Wesmania** [comment]
Yeah, it's fixed. I'll release a stable 0.18.1 with this fix as soon as I'm able.
